src/plugins/vis_type_timeseries/server/lib/vis_data/request_processors/series/date_histogram.js

```diff
@@ -74,7 +74,7 @@ export function dateHistogram(
       ? getDateHistogramForLastBucketMode()
       : getDateHistogramForEntireTimerangeMode();
 
-    // master
+    // cluster_manager
```

Comment:
I have no idea what the purpose of this comment is 😂 - I'd recommend removing as unhelpful